### PR TITLE
Register API, Add id_token input parametr to Google Auth API, Verify register code, PrivacyPolicy Get API, Add Pagination to New list, detail api

### PR DESCRIPTION
**Pull request by sayfullo077**

### CODE REVIEW

**Comment by sayfullo077** [comment]
Fixed

File: account/serializers.py [hunk 1]
```diff
@@ -16,11 +16,22 @@ class Meta:
         fields = ("first_name", "last_name", "email", "password", "device_id")
 
 
+class UserRegisterPhoneSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ("first_name", "last_name", "phone_number", "password", "device_id")
```
Comment (ShukuraliProgrammer) [inline account/serializers.py:22]: keraksiz fieldlarni olib tashlash kerak.

File: account/urls.py [hunk 2]
```diff
@@ -9,14 +9,18 @@
...
     UserRegisterVerifyView,
     UserRegisterView,
+    UserRegisterPhoneView,
+    UserRegisterPhoneVerifyView
 )
 
 urlpatterns = [
     path("register/", UserRegisterView.as_view(), name="register"),
     path("register/verify/", UserRegisterVerifyView.as_view(), name="register-verify"),
+    path("register/phone/<int:phone_number>", UserRegisterPhoneView.as_view(), name="register-phone"),
```
Comment (ShukuraliProgrammer) [inline account/urls.py:19]: endpoint o'zgartirilishi kerak. hamma ma'lumot request body da olinishi kerak

File: account/urls.py [hunk 3]
```diff
@@ -9,14 +9,18 @@
...
+    path("register/verify/<str:code>", UserRegisterPhoneVerifyView.as_view(), name="register-verify-code"),
```
Comment (ShukuraliProgrammer) [inline account/urls.py:20]: bodyda yuborilsin

File: account/urls.py [hunk 4]
```diff
@@ -9,14 +9,18 @@
...
     path("login/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
     path("token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
-    path("google/", GoogleAuth.as_view(), name="googleauth"),
+    path("google/<int:code>", GoogleAuth.as_view(), name="googleauth"),
```
Comment (ShukuraliProgrammer) [inline account/urls.py:23]: code qabul qilinishi shartmas

File: account/utils.py [hunk 5]
```diff
@@ -1,8 +1,13 @@
 import secrets
-
+import requests
 from django.conf import settings
 from django.core.mail import send_mail
+from dotenv import dotenv_values, load_dotenv
+load_dotenv()
+config = dotenv_values(".env")
 
+BOT_TOKEN = config["BOT_TOKEN"]
```
Comment (ShukuraliProgrammer) [inline account/utils.py:9]: bu variable lar settings da yozilishi kerak

File: news/views.py [hunk 6]
```diff
@@ -1,13 +1,39 @@
...
-
 class NewsListView(ListAPIView):
     queryset = News.published.all()
     serializer_class = NewsListSerializer
 
     def get_queryset(self):
         return self.queryset.order_by("-created_at")
+
+
+class NewsListPaginationView(APIView):
```
Comment (ShukuraliProgrammer) [inline news/views.py:18]: NewsListView bilan bitta view bo'lishi kerak

**Pull request merged**